src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -437,4 +437,16 @@
   <data name="DelegateInvocationCanBeSimplified" xml:space="preserve">
     <value>Delegate invocation can be simplified.</value>
   </data>
+  <data name="UseExplicitTyping" xml:space="preserve">
+    <value>Use explicit typing instead of 'var'</value>
```

Comment:
I'd prefer we make a blanket change across the whole code base rather than have a one-off on naming convention for this particular feature. Maybe file an issue to come to agreement on the style and make the change everywhere?